--- ./export.php	2005-07-16 19:24:09.000000000 +0300
+++ /tmp/export.php	2005-07-18 19:28:02.000000000 +0300
@@ -1379,11 +1379,10 @@
 	$numOfGroups = count($group_list);
 	for($counter = 0; $counter < $numOfGroups; $counter++) {
 		$fruity->get_servicegroup_info($group_list[$counter]['servicegroup_id'], $tempGroupInfo);
-		fputs($tempHandler, "define servicegroup{\n");
-		fputs($tempHandler, "servicegroup_name	".$tempGroupInfo['servicegroup_name']."\n");
+		fputs($tempHandler, "define servicegroup {\n");
+		fputs($tempHandler, "\tservicegroup_name\t".$tempGroupInfo['servicegroup_name']."\n");
 		if($tempGroupInfo['alias'] != '')
-			fputs($tempHandler, "alias	".$tempGroupInfo['alias']."\n");
-		fputs($tempHandler, "\n");
+			fputs($tempHandler, "\talias\t".$tempGroupInfo['alias']."\n");
 		fputs($tempHandler, "}\n\n");
 	}
 	fclose($tempHandler);
